### DISCUSSION TITLE
遇到和声就会失真跑调，如何解决？

### DISCUSSION THREAD

**Discussion by JerryKingQAQ** [svc-develop-team/so-vits-svc#326] [Q&A]
如题，我自己用UVR提取歌曲中的人声依然有一定的和声存在。训练的模型一旦进行和声部分就会出现失真、电音、跑调的问题？我想请教下大家是如何解决的。

**Comment by bfloat16** [comment]
可以尝试使用UVR里面的5_HP-Karaoke-UVR.pth去除和声，具体参数如图
<img width="803" alt="屏幕截图 2023-07-22 150513" src="https://github.com/svc-develop-team/so-vits-svc/assets/38366253/a8e83780-7835-4114-913e-8c21b5973fde">
